### DISCUSSION TITLE
NFS ProStreet sound issues (FMV fine, but music and audio in-game is sped up/skipping)

### DISCUSSION THREAD

**Discussion by BenneyR** [xenia-canary/game-patches#194] [Trash]
I recently got NFS ProStreet, as well as Carbon and Most Wanted (haven't started them yet) and I am having an audio bug.

Everything else in the game is running perfectly, except for the in-game audio outside of cutscenes/FMV footage.

**It is basically going at super speed, playing all the songs and dialogue audio very quickly and also skipping around a lot while doing it.**

What is weird is the cars seem perfectly fine, so only the announcers/music are messing up.

**Does anyone know what settings the config should be set to avoid this problem?**

_I was able to get Horizon 1 to run flawlessly, but there are guides for that game to follow so that makes sense._

**Comment by Margen67** [comment]
Unless you're asking for a patch for this game, ask for help on the [Xenia Discord server](https://discord.gg/Q9mxZf9)

**Reply by BenneyR** [reply]
Oh I didn't know there was a Discord Server!

That'll help a ton, thank you!
